sonar/modules/config.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 RERO.
+#
+# Swiss Open Access Repository is free software; you can redistribute it
+# and/or modify it under the terms of the MIT License; see LICENSE file for
+# more details.
+
+"""SONAR specific configuration."""
+
+SONAR_FIXTURES_API_URL = 'https://127.0.0.1:5000/api/'
```

Comment:
why not localhost?

Reply:
I think 127.0.0.1 always exists, it's not the case for localhost. But, as discussed, we keep localhost for consistency with invenio